src/interfaces/IStandardExecutor.sol

```diff
@@ -25,4 +25,14 @@ interface IStandardExecutor {
     /// @param calls The array of calls.
     /// @return An array containing the return data from the calls.
     function executeBatch(Call[] calldata calls) external payable returns (bytes[] memory);
+
+    /// @notice Execute a call using a specified runtime validation, as given in the first 21 bytes of
+    /// `authorization`.
+    /// @param data The calldata to send to the account.
+    /// @param authorization The authorization data to use for the call. The first 21 bytes specifies which runtime
+    /// validation to use, and the rest is sent as a parameter to runtime validation.
+    function executeWithAuthorization(bytes calldata data, bytes calldata authorization)
```

Comment:
I'm curious to hear your thoughts on why you prefer the new function to be part of the standard executor rather than an extension executor :)

Reply:
To be honest I don't have a strict reasoning - I think we could split it out, or we could try to bring together all of the required account functions into one interface for simplicity. I think I'm leaning more towards the latter to make it easier to read the spec, but open to discussing.

Reply:
I am more lean towards to split it out. 
StandardExecutor indicates the functions here are following a certain standard (ERC-4337).

Reply:
Technically the functions in `IStandardExecutor` are only standardized in ERC-6900, but 6900 also requires the plugin installation functions, which are in a separate interface `IPluginManager`.

As for `executeWithAuthorization`, it's something that all 6900 compliant accounts will need to implement after this spec change. Do y'all have any suggestions for what to name an interface that only holds this one function? Or how to rename `IStandardExecutor` or other existing account interfaces to hold it?

Reply:
Actually to rethink on this, especially since executeWithAuthorization is something that all 6900 compliant accounts will need to implement. We can just keep it in there.